addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -23,14 +22,21 @@ class AccountMove(models.Model):
 
     l10n_it_edi_transaction = fields.Char(copy=False, string="FatturaPA Transaction")
     l10n_it_edi_attachment_id = fields.Many2one('ir.attachment', copy=False, string="FatturaPA Attachment", ondelete="restrict")
-
     l10n_it_stamp_duty = fields.Float(default=0, string="Dati Bollo", readonly=True, states={'draft': [('readonly', False)]})
-
     l10n_it_ddt_id = fields.Many2one('l10n_it.ddt', string='DDT', readonly=True, states={'draft': [('readonly', False)]}, copy=False)
-
     l10n_it_einvoice_name = fields.Char(compute='_compute_l10n_it_einvoice')
-
     l10n_it_einvoice_id = fields.Many2one('ir.attachment', string="Electronic invoice", compute='_compute_l10n_it_einvoice')
+    l10n_it_amount_split_payment = fields.Monetary(compute="_compute_l10n_it_amount_split_payment")
```

Comment:
... are we sure we don't need it in a view, just like we have for Withholding and Pension Fund amount?

Reply:
No, but you did not add it.  And do not we kind of have it already through the tax group?  (It does not seem that important in a tree view of invoices)

Reply:
I didn't yet add it... but I don't know, maybe they want to see why that invoice has zero VAT because of split payment... ? 
I'm asking, I'm not sure.
In the case of Withholding, they have to consider it because they have to pay to the Tax Agency.
In case of Split payment... they don't have to do much with that amount. 🤷🏻 

Think I'll make it a function following your suggestion.